src/components/CommonTextStyles/variables.js

```diff
@@ -0,0 +1,29 @@
+//colors
+export const palette = {
+  purple: '#4A2ACB',
+  darkpurple: '#37119D',
+  palepurple: '#E5E4FF',
+  bluegrey: '#4C6282',
+  blue2: '#43BFFF',
+  darkblue: '#1C325D',
+  lightblue: '#CDE8F8',
+  whiteblue: '#E4F3FB',
+  footerblue: '#01A5EC',
+  orange: '#F0BA8C',
+  warningred: '#972232',
+  white: '#fff',
+};
```

Comment:
you can create a type Color 

```javascript
type Color = keyof typeof palette
```

then we have Color-key as anything in the palette

Reply:
That Color could be then the prop in the Text.tsx and the color of the Text could be mapped using this palette

Reply:
And lol, why is this file a .js file?